packages/libs/components/src/map/animation_functions/geohash.tsx

```diff
@@ -33,10 +33,9 @@ export default function geohashAnimation({
      **/
     zoomType = 'in';
     const hashDif = currentGeohash.length - prevGeoHash.length;
-    // Get a new array of new markers with existing position property
-    // Set final render markers to the cloneArray which holds the new markers with
-    // their new starting location
-    consolidatedMarkers = updateMarkers(markers, prevMarkers, hashDif);
+    // Get array of new markers with old positions
+    const cloneArray = updateMarkers(markers, prevMarkers, hashDif);
+    consolidatedMarkers = [...prevMarkers, ...cloneArray];
```

Comment:
Reversing the order of the spreads here does not fix the "old marker hidden under new markers" problem.